### PR TITLE
explain how to run tests for a given plugin

### PR DESCRIPTION
**Pull request by dachary**
See
https://meta.discourse.org/t/how-do-i-run-only-discourse-plugins-poll-spec/47244/5
for the corresponding discussion.

Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

File: docs/VAGRANT.md [hunk 1]
```diff
@@ -118,6 +118,18 @@ RAILS_ENV=test bundle exec rake db:migrate
 bundle exec rake autospec p l=5
 ```
 
+When developing a plugin (the *poll* plugin for instance), the following can be used to only run all tests from the *spec* directory:
+
+```
+RAILS_ENV=test bundle exec rake plugin:spec poll
```
Comment (tgxworld) [inline docs/VAGRANT.md]: `RAILS_ENV` is not needed here.

Comment (dachary) [inline docs/VAGRANT.md]: removed, thanks ! I'm slightly confused about how isolated the est contexts are :-P https://meta.discourse.org/t/running-tests-breaks-the-dev-environment/47434

File: docs/VAGRANT.md [hunk 2]
```diff
@@ -118,6 +118,18 @@ RAILS_ENV=test bundle exec rake db:migrate
...
+When developing a plugin (the *poll* plugin for instance), the following can be used to only run all tests from the *spec* directory:
+
+```
+bundle exec rake plugin:spec poll
+```
+
+or just a single file with:
+
+```
+RAILS_ENV=test LOAD_PLUGINS=1 bundle exec rspec plugins/poll/spec/controllers/posts_controller_spec.rb
```
Comment (tgxworld) [inline docs/VAGRANT.md]: Same for this line as well :)

Comment (dachary) [inline docs/VAGRANT.md]: done and repushed !

**Comment by tgxworld** [comment]
Thank you!!!

**Pull request merged**